__test__/ship.test.js

```diff
@@ -0,0 +1,83 @@
+const Ship = require('../src/ship')
+const Port = require('../src/port')
+const Itinerary = require('../src/itinerary')
```

Comment:
These tests are for testing the `Ship` constructor. I would recommend trying to use mock data to replace `Port` and `Itinerary`. You've already done this for most tests in this folder, you only need to change it for the last test